eventonica-frontend/src/components/AddEvent.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+
+export default function AddEvent(props) {
```

Comment:
You can desructure the props here to make this a little cleaner, so delete line 4 and write:

```
export default function AddEvent({state, dispatch, handleAddEventForm}) {
  ...
```